<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="PDCDT01GY4ZVQ1" ignored="0" name="Test" tests="8" failures="1" timestamp="2020-02-11T07:25:20 GMT" time="191.526" errors="0">
  <testcase name="ScenarioFour" time="2.995" classname="com.amazon.testpage.RegistrationTest"/>
  <testcase name="ScenarioOne" time="29.616" classname="com.amazon.testpage.RegistrationTest"/>
  <testcase name="ScenarioFive" time="3.996" classname="com.amazon.testpage.LoginTest"/>
  <testcase name="ScenarioFour" time="0.874" classname="com.amazon.testpage.LoginTest"/>
  <testcase name="ScenarioOne" time="23.43" classname="com.amazon.testpage.LoginTest"/>
  <testcase name="FirstTest" time="10.985" classname="com.amazon.testpage.CreateWishList"/>
  <testcase name="SecondTest" time="3.773" classname="com.amazon.testpage.CreateWishList"/>
  <testcase name="FourthTest" time="22.995" classname="com.amazon.testpage.CreateWishList">
    <failure type="java.lang.AssertionError" message="expected [Private!!!] but found [Private]">
      <![CDATA[java.lang.AssertionError: expected [Private!!!] but found [Private]
at com.amazon.testpage.CreateWishList.FourthTest(CreateWishList.java:154)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- FourthTest -->
</testsuite> <!-- Test -->
